src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,179 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final String INDEX_OUT_OF_BOUNDS = "Index is out of bounds of list";
+    private ListNode<T> head;
+    private ListNode<T> tail;
+    private int size = 0;
```

Comment:
int has default value 0
```suggestion
    private int size;
```